packages/react/src/useUploadThing.ts

```diff
@@ -85,6 +86,7 @@ export const INTERNAL_uploadthingHookGen = <
       opts?.onUploadProgress?.(0);
       try {
         const res = await uploadFiles<TEndpoint, TSkipPolling>(endpoint, {
+          signal: opts?.signal,
```

Comment:
maybe rename this to abortSignal? I could imagine this being confusing with such a generic name

Reply:
hmm `fetch` takes `signal`, I feel like it's quite a well-used term especially when it comes to interruption? i guess it can be confused with reactive signals though 🤔

Reply:
Yeah, that's what I was thinking. Maybe it is ok though since it is what `fetch` uses